app/views/menu/help-menu.tmpl.html

```diff
@@ -0,0 +1,21 @@
+<span role="help-button">
```

Comment:
Overall, I made a comment in the original issue, I don't think that we should have 3 entries with the word `RAML` in them this is now:

- RAML Specification
- RAML Website
- Report a Bug
- RAML Tutorial

2 things:
 Remove RAML from the items
 Do Not Capitalize Every Word ;)

- About RAML -> Link to website
- Language spec -> Link to Specification
- Tutorial -> Link to tutorial
- Report a bug -> Link to issues

WDYT?